components/frontend/src/dashboard/FilterCard.css

```diff
@@ -0,0 +1,16 @@
+.ui.card.filter_card .table {
+    margin-top: 0.1em;
+    overflow: hidden;
+    white-space: nowrap;
+    table-layout: fixed;
+}
+
+.ui.card.filter_card .header {
+    margin-bottom: 0.3em;
+    overflow: hidden;
+    white-space: nowrap;
+}
+
+.ui.card.filter_card .header.blue {
+    color: #2185d0;
```

Comment:
Do we not have one css file with all colour definitions?

Reply:
No, but good suggestion. I'll add one.

Reply:
I've added a few color variables to `App.css`, the rest we can do later.